native/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Super POM included by all other sub-POM(s)-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>native</artifactId>
+    <packaging>pom</packaging>
+    <name>net.java.dev.jna:native</name>
+
+	<parent>
+		<groupId>net.java.dev.jna</groupId>
+		<artifactId>parent</artifactId>
+		<version>4.3.0-SNAPHSOT</version>
+		<relativePath>../parent</relativePath>
+	</parent>
+	
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
```

Comment:
True, but it helps since there is an older "groupId" that is sometimes mentioned in some references _org.apache.maven.plugin_ (i.e., without the 's') so it helps to know where to look for the plugin.